### PR TITLE
early feedback for ToTV provider

### PR DESCRIPTION
**Pull request by Terebi42**
Not asking for this to be committed yet, just looking for feedback.

### CODE REVIEW

File: src/NzbDrone.Core/Indexers/TitansOfTv/TitansOfTvSettings.cs [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        {
+          //  RuleFor(c => c.ApiKey).ValidRootUrl();
+        }
+    }
+
+    public class TitansOfTvSettings : IProviderConfig
+    {
+     private static readonly TitansOfTvSettingsValidator Validator = new TitansOfTvSettingsValidator();
+
+     public TitansOfTvSettings()
```
Comment (markus101) [inline src/NzbDrone.Core/Indexers/TitansOfTv/TitansOfTvSettings.cs]: This constructor isn't required (ApiKey would just be blank).

File: src/NzbDrone.Core/Indexers/TitansOfTv/TitansOfTvSettings.cs [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+        {
+            ApiKey = "";
+        }
+
+        [FieldDefinition(0, Label = "API key", HelpText = "Enter your ToTV API key. (My Account->API->Site API Key)")]
```
Comment (markus101) [inline src/NzbDrone.Core/Indexers/TitansOfTv/TitansOfTvSettings.cs]: Could add a `HelpLink` that links them right to the API key.

**Comment by markus101** [comment]
Looks pretty good at first glance, some formatting issues (extra new lines and such) that need to be cleaned up. The biggest issue is the merged PR from Sonarr/Sonarr into your branch. you need to rebase upstream changes into your branch, never merge. That will need to be fixed before this PR could be accepted (and the relevant commits squashed into one).

File: src/NzbDrone.Core/Indexers/TitansOfTv/TitansOfTvSettings.cs (hunk 1 above)
Comment (Terebi42) [inline src/NzbDrone.Core/Indexers/TitansOfTv/TitansOfTvSettings.cs]: Ya, didn't know if they would be needed for serialization or something. Ill remove it

**Comment by level42ca** [comment]
Is there any update on this? Was this ever merged into Sonarr:Develop?

**Comment by markus101** [comment]
@level42ca No, still open = not in develop.

@Terebi42 are you going to open a new PR for ToTV? Just saw that the link to the old repo is dead (PR is coming from Unknown Repo), but there is a repo there. Let us know if we should close this and if you need anything to get it finished, would love to get this merged.

**Comment by Terebi42** [comment]
@markus101 Im hoping to put it in this weekend, but I think I might need some hand holding doing the rebase/squish. I'm from a TFS background, so I have no idea what I'm doing with that part.  Ill hit you up on IRC

**Comment by needo37** [comment]
Is there any thing us users can do to help with this PR?

**Comment by SAKUJ0** [comment]
@needo37 
I am trying to build it in right now, however I don't have Visual Studio and I am messing with the other PR that apparently allows for Sonarr to be built on Linux.

**Comment by markus101** [comment]
I opened a new clean PR: https://github.com/Sonarr/Sonarr/pull/639

**Pull request closed without merging**